src/components/Table/index.js

```diff
@@ -23,34 +26,68 @@ export default class Table extends Component {
         }
     }
 
+    onColumnSelect(e, columnIndex) {
+        const { onSort } = this.props;
+        onSort(e, this.state.columns[columnIndex].field);
+        this.setState({ selectedColumn: columnIndex });
+    }
+
+    getContainerClassNames() {
+        const { className } = this.props;
+        return classnames('rainbow-table-wrapper', className);
+    }
+
     resolveColumnsFomChilren() {
         const { children } = this.props;
         const newColumns = resolveColumns(children);
         this.setState({ columns: newColumns });
     }
 
-    render() {
-        const { data, className, style } = this.props;
+    resize(event, columnIndex, width) {
         const { columns } = this.state;
+        const resizedColumn = Object.assign({}, columns[columnIndex]);
+        resizedColumn.width = width;
+        columns[columnIndex] = resizedColumn;
+        this.setState({ columns });
```

Comment:
Here when you resize the table is completely rerendered, we need to find a way to optimize this. I think you can change only the width of the header column without recreate the table.